introduction.md

```diff
@@ -5,3 +5,8 @@ golang and git are awesome so far, thanks for your effort.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?